runtime/drivers/olap.go

```diff
@@ -326,6 +326,9 @@ func (d Dialect) AutoUnnest(expr string) string {
 }
 
 func (d Dialect) MetricsViewDimensionExpression(dimension *runtimev1.MetricsViewSpec_Dimension) string {
+	if dimension.LookupTable != "" {
+		return d.DictGetExpr(dimension.LookupTable, dimension.LookupValueColumn, dimension.Column)
+	}
 	if dimension.Expression != "" {
 		return dimension.Expression
```

Comment:
1. Can it also support `dimension.Expression` here in addition to `dimension.Column`? Sometimes people use `expression` instead of `column:`, like `expression: country`.
2. The `DictGetExpr` returns `""` for DuckDB, Druid and Pinot, which might give weird errors. Can you think of some safer handling, e.g. a simple sub-select or returning an error?

Reply:
I think it'd be good to support `expression` for consistency. If they use `dictGet` in the expression, they should also get a validation error right? (Due to nested `dictGet`)